### DISCUSSION TITLE
How to Git?

### DISCUSSION THREAD

**Discussion by Nikhil8557** [community/community#46512] [New to GitHub] [answered]
How to create repository?

**Comment by Nikhil8557** [comment, marked as answer]
Check Google and YouTube

**Comment by queenofcorgis** [comment]
Our docs can help! [Create a repository](https://docs.github.com/en/get-started/quickstart/create-a-repo) is a great starting point 😄

**Comment by Juanescacha** [comment]
You first need to understand the difference with Git and Github, and being that said you can follow as you been told to check the docs, you only go to here: 
![image](https://user-images.githubusercontent.com/6394715/218217547-054efc91-4056-41c9-bea2-75ab96ed1930.png)
create a new Repository 
and follow the instructions that are quit easy to follow and that would be all, as that simple!
